packages/javascript-sdk/src/config/interfaces.ts

```diff
@@ -29,7 +29,7 @@ interface ConfigOptions {
   redirectUri?: string;
   scope?: string;
   serverConfig?: ServerConfig;
-  tokenStore?: TokenStoreObject | 'indexedDB' | 'sessionStorage' | 'localStorage';
+  tokenStore?: TokenStoreObject | 'sessionStorage' | 'localStorage';
```

Comment:
i'm curious if we think this is non releasing change. I'm not enitrely sure what it should be considered, but it feels like its worh putting as a minor. Even though its a part of the breaking change release, we should probably release this with `beta` tag anyways.

Reply:
Yeah, honestly, I'm not sure if this is technically a breaking change. It is from a developer perspective since it removes a configuration option. If they are expecting tokens to be in IndexedDB (maybe they directly access them?), and then we remove them. We could break their codebase. But, maybe I'm stretching it?

Reply:
I see you included the `!` so that should indicate a release, i missed that on the initial pass. Yeah, im not really sure what to consider it but i'm happy as long as we trigger a release for this change